specs/latest/1.0/index.html

```diff
@@ -1979,6 +1980,20 @@ <h4>Attributes</h4>
             The actual height of the drawing buffer. May be different from the
             <code>height</code> attribute of the <code>HTMLCanvasElement</code> if
             the implementation is unable to satisfy the requested width or height.
+        <dt>
+            <code class=attribute-name>
+                <a id="DOM-WebGLRenderingContext-colorSpace">
+                    colorSpace
+                </a>
+            </code>
+            of type <code><a href="https://html.spec.whatwg.org/multipage/canvas.html#predefinedcolorspace">PredefinedColorSpace</a></code>
+            <a href="#refsPREDEFINEDCOLORSPACE">(specification)</a>
+        <dd>
+            The color space in which the drawing buffer's contents are assumed to
+            have been produced. Changing this attribute affects the on-screen display
+            of the rendering context, but does not change the drawing buffer's
```

Comment:
Does it also change the color space tag of this context's canvas when used as a source for other calls (like drawImage, texImage2D on another context, etc.)?

Reply:
Good point - yes, it does change that behavior. Adding that in the next follow-up.

Reply:
Added now.